index.html

```diff
@@ -53,6 +48,171 @@ <h2>I am a Software Developer</h2>
                     </ul>
                 </div>
             </section>
+             <!-- =============
+            Recent Works section
+        =========== -->
+        <section class="recent-works">
+            <h3>My Recent Works</h3>
+            <hr>
+            <div class="recent-work">
+                <a href=""><img src="./image/Img_Placeholder_1.png" alt="Women doing yoga"></a>
+                <h4>Multi-Post Stories</h4>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                <ul class="tech-used">
+                    <li>CSS</li>
+                    <li>html</li>
+                    <li>bootsrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="project-button " type="button">See Project</button>
+            </div>
+            <div class="recent-work">
+                <a href="#"><img src="./image/Placeholder12.png" alt="Women doing yoga"></a>
+
+                <button class="project-button " type="button">See Project</button>
+            </div>
+            <div class="cards-section">
+                <div class="work-card">
+                    <div class="card-availability">
+                        <h4>Data Dashboard Healthcare</h4>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                        <ul>
+                            <li>html</li>
+                            <li>bootsrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button type="button">See Project</button>
+                </div>
```

Comment:
There is an issue with the display of your buttons as you can see in the following image..

![image](https://user-images.githubusercontent.com/15807343/149385439-377c0715-7d7d-4a40-9c1d-29a13e157494.png)

#### Please fix this issue so that your works section looks as expected 🔥🤓 
Hint: You've done a great job using flexbox to align your techs but the width of your `ul` tag seems to be causing the issue